### PR TITLE
Docs: Edit 2.2.0 docs

### PR DESCRIPTION
**Pull request by kirszenbaum**
This PR:
- Edits the 2.2.0 changelog
- Edits the 2.2.0 release notes
- Edits the new `i18n-overview.md` guide
- Edits the new `importing-files.md` guide
- Makes other minor edits

### CODE REVIEW

File: docs/guide/release-notes.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ HyperFormula adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
...
+- Added an American English (`enUS`) language pack. It's a convenience alias: it contains the same translations as the existing British English (`enGB`) language pack. [#1025](https://github.com/handsontable/hyperformula/issues/1025)
 
 ### Fixed
-- Fixed functions VLOOKUP and HLOOKUP to handle duplicates according to the specification. [#1072](https://github.com/handsontable/hyperformula/issues/1072)
-- Fixed function MATCH to handle ranges ordered descending according to the specification. [#1063](https://github.com/handsontable/hyperformula/issues/1063)
+- Fixed functions VLOOKUP and HLOOKUP to handle duplicates in the way specified by the [OpenDocument](https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part4-formula/OpenDocument-v1.3-os-part4-formula.html#HLOOKUP) standard. [#1072](https://github.com/handsontable/hyperformula/issues/1072)
+- Fixed the MATCH function to handle descending ranges in the way specified by the [OpenDocument](https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part4-formula/OpenDocument-v1.3-os-part4-formula.html#MATCH)
```
Comment (sequba) [inline docs/guide/release-notes.md:15]: Missing link to the issue

**Comment by kirszenbaum** [comment]
@sequba – I gathered my propositions in this PR. I've got two further comments:

- The **Localizing functions** guide doesn't actually say how to add another translation set / language pack.
- We could use an example that shows integration with a CSV parser library (analogically to the ExcelJS example)

**Comment by sequba** [comment]
> * The **Localizing functions** guide doesn't actually say how to add another translation set / language pack.
> * We could use an example that shows integration with a CSV parser library (analogically to the ExcelJS example)

Good point:
- https://github.com/handsontable/hyperformula/issues/1103
- https://github.com/handsontable/hyperformula/issues/1104

File: docs/guide/file-import.md [hunk 2]
```diff
@@ -1,21 +1,24 @@
-# Importing XLSX/CSV files into HyperFormula
+# File import
 
-Use HyperFormula to calculate the formulas exported from other spreadsheets.
+Import XLSX and CSV files into HyperFormula.
 
 ## Overview
 
-HyperFormula itself has no file import functionality. Rather, the [factory methods](../api/classes/hyperformula.md#factories) use standard JavaScript data types for easy integration with any data importing solution.
+HyperFormula has no built-in file import functionality. But its [factory methods](../api/classes/hyperformula.md#factories) use standard JavaScript data types. This lets you easily integrate HyperFormula with any third-party file import library.
```
[the remark this answers is not in the document]
Comment (kirszenbaum) [reply docs/guide/file-import.md]: Got it, how about this then: [ec8d791](https://github.com/handsontable/hyperformula/pull/1099/commits/ec8d7919dd468a2867fa9fe39869dcd2c9d1a115) ("solutions" sounds like we're talking about products)

File: docs/guide/i18n-features.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+```
+
+To use the US date and time formats, set:
+
+```javascript
+dateFormats: ['MM/DD/YYYY', 'MM/DD/YY', 'YYYY/MM/DD'], // US date formats
+timeFormats: ['hh:mm', 'hh:mm:ss.sss'], // set by default
+```
+
+You can also add custom ways of [handling dates and times](date-and-time-handling.md#custom-date-and-time-handling).
```
[the remark this answers is not in the document]
Comment (kirszenbaum) [reply docs/guide/i18n-features.md]: From our perspective, that's right. From the user's perspective, though, the information structure is not consistent: some settings are listed on the page, but for other ones, you need to click around to find a simple code sample, and then go back to continue with the setup.

Ideally, we could use some content reuse here. But if maintainability is our priority, we can stay with the links: [cd7483a](https://github.com/handsontable/hyperformula/pull/1099/commits/cd7483af08d54f1af6f7535699abfd9c6050e4f8)

File: docs/guide/file-import.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sequba) [reply docs/guide/file-import.md]: Thanks. That sounds good.

**Review by sequba — APPROVED** [review]

File: docs/guide/i18n-features.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sequba) [reply docs/guide/i18n-features.md]: I consulted it with @warpech and we decided to stay with the links. Thank you @kirszenbaum

**Pull request merged**